frontend/test/cypress/integration/product-list/filters.spec.ts

```diff
@@ -93,13 +85,13 @@ describe('product list filters', () => {
          user.findByText(/device:\s*iphone/i).should('exist');
       });
 
-      waitForSearchCompletion();
+      user.wait('@search');
 
       user
          .findByRole('button', { name: /collapse device$/i, expanded: true })
          .click(getVirtualListClickOptions());
 
-      user.wait(500);
+      user.wait(300);
```

Comment:
Why are we waiting for only 300ms here instead of 500ms?

Reply:
I'm trying to reduce the waiting amount so that we spare a little bit of time.. I'd like to improve the way this is handled eventually